string objectname
string mesh_filepath
geometry_msgs/PoseStamped pose_stamped